frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -271,7 +272,29 @@ function ReaderWikipedia:addToMainMenu(menu_items)
                         end,
                     })
                 end,
-            }
+                separator = true,
+            },
+            { -- setting used in wikipedia.lua
+                text = _("Show page image"),
```

Comment:
I find this somewhat unclear, though I'm struggling to come up with something better.